pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunctionTest.java

```diff
@@ -116,6 +145,90 @@ public void testCaseTransformFunctionWithIntResults() {
     }
   }
 
+  @Test
+  public void testCaseTransformFunctionWithoutCastForFloatValues() throws Exception {
```

Comment:
(formatting) Can you reformat the changes with [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#set-up-ide)

Reply:
resolved